src/sage/geometry/hyperplane_arrangement/arrangement.py

```diff
@@ -705,17 +714,18 @@ def plot(self, **kwds):
 
     def cone(self, variable='t'):
         r"""
-        Return the cone over the hyperplane arrangement.
+        Return the cone over the hyperplane arrangement `H_1,\dots,H_n`.
```

Comment:
This change provides no new information, and on the contrary, actually adds some ambiguity (it suggests that $H_1, \ldots, H_n$ should be input; also `\dots` is evil IMO).